<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;
QMainWindow::titleBarMaxButton {
    background: transparent;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>340</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>340</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white;
    color: black;
}
QPushButton:hover {
    background-color: lightgray;
    color: black;
}
QPushButton::item:selected {
    background-color: #0078d7;
    color: white;
    selection-background-color: #a5c8f7;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white;
    color: black;
}
QPushButton:hover {
    background-color: lightgray;
    color: black;
}
QPushButton::item:selected {
    background-color: #0078d7;
    color: white;
    selection-background-color: #a5c8f7;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white;
    color: black;
}
QPushButton:hover {
    background-color: lightgray;
    color: black;
}
QPushButton::item:selected {
    background-color: #0078d7;
    color: white;
    selection-background-color: #a5c8f7;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>340</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label2_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>340</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label3_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>320</y>
      <width>340</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label1_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>340</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label2_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>200</y>
      <width>340</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label3_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>320</y>
      <width>340</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label1_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>340</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>10</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white;
    color: black;
}
QPushButton:hover {
    background-color: lightgray;
    color: black;
}
QPushButton::item:selected {
    background-color: #0078d7;
    color: white;
    selection-background-color: #a5c8f7;
}</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>340</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>440</y>
      <width>340</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>440</y>
      <width>340</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;</string>
   </property>
   <widget class="QMenu" name="menuDifficulty">
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QMenu {
    background-color: white;
    color: black;
}
QMenu::item:selected {
    background-color: #0078d7;
    color: white;
    selection-background-color: #a5c8f7;
}</string>
    </property>
    <property name="title">
     <string>Difficulty</string>
    </property>
    <addaction name="action1"/>
    <addaction name="action2"/>
    <addaction name="action3"/>
    <addaction name="action4"/>
   </widget>
   <addaction name="menuDifficulty"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
